package.json

```diff
@@ -102,4 +102,4 @@
       "eslint"
     ]
   }
-}
\ No newline at end of file
+}
```

Comment:
Strange that we get commits such as these =/ Should not happen with proper linting.